apps/dashboard/forms.py

```diff
@@ -20,12 +20,19 @@ class DashboardProjectCreateForm(ProjectCreateForm):
 
     class Meta:
         model = project_models.Project
-        fields = ['name', 'description', 'is_public']
+        fields = ['name', 'description', 'access']
         widgets = {
-            'is_public': forms.RadioSelect(
+            'access': forms.RadioSelect(
                 choices=[
-                    (True, _('All users can participate (public).')),
-                    (False, _('Only invited users can participate (private).'))
+                    (project_models.Access.PUBLIC.value,
+                     _('All users can see project tile and content and can '
+                       'participate (public).')),
+                    (project_models.Access.SEMIPUBLIC.value,
+                     _('All users can see project tile and content, only '
+                       'invited users can participate (semi-public).')),
+                    (project_models.Access.PRIVATE.value,
+                     _('Only invited users can see project tile and content '
+                       'and can participate (private).'))
                 ]
-            )
+            ),
```

Comment:
So cant we remove the labels here then?

Reply:
I'd prefer to leave this in as it's actually how it should be done - the overwrite is just a weird workaround IMO. I'd really like to get rid of it again :/

Reply:
Alright, I understand. But then maybe a note that this is actually not used for the moment? Because it might be confusing/not so transparent for someone who doesnt know the code..

Reply:
Right, will add

Reply:
Done

Reply:
Thank you!